src/apps/btc/btc.c

```diff
@@ -49,6 +52,22 @@ bool app_btc_xpub(
     return btc_common_encode_xpub(&derived_xpub, xpub_type, out, out_len);
 }
 
+bool app_btc_identifier_xpub(const uint32_t* keypath, size_t keypath_len, char* out, size_t out_len)
+{
+    if (keypath_len != 2) {
+        return false;
+    }
+    if (keypath[0] != BTC_TOP_IDENTIFIER && keypath[1] != BTC_BOTTOM_IDENTIFIER) {
```

Comment:
this check does not what you want :D (if you check the test coverage report, you will see the bug ;))

Reply:
Ah damn it ^^